demos/common/samples/common.hpp

```diff
@@ -412,60 +412,40 @@ static UNUSED bool writeOutputBmp(std::string name, unsigned char *data, size_t
 * @param width - width of the rectangle
 * @param rectangles - vector points for the rectangle, should be 4x compared to num classes
 * @param classes - vector of classes
-* @param thickness - thickness of a line (in pixels) to be used for bounding boxes
 */
-static UNUSED void addRectangles(unsigned char *data, size_t height, size_t width, std::vector<int> rectangles, std::vector<int> classes, int thickness = 1) {
+static UNUSED void addRectangles(unsigned char *data, size_t height, size_t width, std::vector<int> rectangles, std::vector<int> classes) {
```

Comment:
May be `object_detection_demo_faster_rcnn` could just use OpenCV? And the need in this function would be eliminated. :)

Reply:
Fair point. I'll see if it's not too difficult to port this demo to OpenCV.

Reply:
Ported it in #606. This change no longer applies, so I removed it.